### ISSUE TITLE
[Bug Report][3.5.16] VSelect always jumps back to the first record when making multiple selections

### ISSUE THREAD

**Issue by AndyBeat** [vuetifyjs/vuetify#19635]
### Environment
**Vuetify Version:** 3.5.16
**Vue Version:** 3.4.21
**Browsers:** Chrome 122.0.0.0
**OS:** Windows 10

### Steps to reproduce
Choose item[Alabama], then item[Guam]

### Expected Behavior
Same effect as Vuetify3.5.15 version

### Actual Behavior
Jump to Alabama when choosing Guam

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNplVFFPGzEM/ivRvWyTCtU0bQ+oQ0IgoGMFRNnQBDyYO7dnXS45xUmhQ/z3OUlLe91T7C/258+O7+5fC3bl8Kjr9hcBi4Ni5LHtNHg8fDBKjRZ7jBpLHx2lFnutrVB/fyhmsLCOPPJDka8OxGlZbthL8jtck/ECXlPZqKUNTq0TVT9Ow1PineZqK7QN2lOnMXsdOib2aPxepI3g4Wi4Vih6R8Mt8eJy6aiLF0rhS2edVxXOQDjVa2aswIP6+GntKuXQB2c2vnrXywfq/nGwwbN8ATeQUh+OpA9o4cMgmdxkq0VHJRg1hdZmxNFfa8TsJ7sGDAPHiGPQNLPOUIo/tto6qGy2jZF2qQx+J/8ENTyDwxh1IoOSol7ZmZLs0D5lplOs0InySk1TA/F+QqWzBjlG9AhPpSpVKe8MrZtnirMAbTzPpRhRtMYV1KKtlzvWmoyl1MzYVASxXTHtczov3ju9kPcMZbPcyf9pA4minDUBMqmtCTiuQWs1Zg2mSgSCLaOzQzABZijrwOh9jqOypjmYbMsQ2frMTix7xdR10k2fQ2BZ2tTkxBq/knOJT279uJe4gDiiXt4lPqtzaDuuKT9HBH7I+qK0mb0JvlCZHjR6f6xrdjlkX2t1DM7KIHOphJxAs9KdfJR1lQnESW0P5aqm3Re5arQ8U17OK4dzmyZxDRpCMtAYXuoFrHbuOqDzVt0klT2im1p+AqtiMXJqw47SjGyU3go3MqPMosd0iy95CX55qOP5G10rc04mycKZrTIZ+G9J70CmbOY+t3OH7NUmUgDi0hqmfLu0rcRuM2y+6bdsvCVAnNFw9f8o3gYmaD0ovux/3f/8rXj8BwO9krg=)

### Other comments
Affected by these 2 issues #19593 and #19574..

**Comment by AndyBeat** [comment]
@yuwu9145 You seem to have misunderstood what I meant.  Instead of always scrolling to the last one, it will stay at the selected one, just like the version 3.5.15.
For example, select [Alabama], then [Guam], and then select [Colorado] instead of jumping back to [Guam], you stay in [Colorado].

**Comment by yuwu9145** [comment]
@AndyBeat Yes, the first fix [d65d586](https://github.com/vuetifyjs/vuetify/commit/d65d5865af5e13d45f5f9c00b380504ad44f5144) was wrong.

There is a new fix #19642 that makes sure stay at/scroll to "the last selected one". I plan to do more testing around this change over weekend

**Comment by AndyBeat** [comment]
@yuwu9145 OK, thanks
